donkeycar/parts/telemetry.py

```diff
@@ -15,31 +14,40 @@
 from paho.mqtt.client import Client as MQTTClient
 
 
-class Telemetry(StreamHandler):
-    '''
+class MqttTelemetry(StreamHandler):
+    """
     Telemetry class collects telemetry from different parts of the system and periodically sends updated to the server.
     Telemetry reports are timestamped and stored in memory until it is pushed to the server
-    '''
+    """
 
-    def __init__(self, cfg, default_inputs = [], default_types = []):
+    def __init__(self, cfg, default_inputs=None, default_types=None):
 
         StreamHandler.__init__(self)
 
         self.PUBLISH_PERIOD = cfg.TELEMETRY_PUBLISH_PERIOD
         self._last_publish = time.time()
         self._telem_q = queue.Queue()
-
-        self._default_inputs = default_inputs
-        self._default_types = default_types
+        self._default_inputs = default_inputs if default_inputs is not None else []
+        self._default_types = default_types if default_types is not None else []
         self._total_updates = 0
         self._donkey_name = os.environ.get('DONKEY_NAME', cfg.TELEMETRY_DONKEY_NAME)
-        self._mqtt_broker = os.environ.get('DONKEY_MQTT_BROKER', cfg.TELEMETRY_MQTT_BROKER_HOST) # 'iot.eclipse.org'
+        self._mqtt_broker = os.environ.get('DONKEY_MQTT_BROKER', cfg.TELEMETRY_MQTT_BROKER_HOST)  # 'iot.eclipse.org'
         self._topic = cfg.TELEMETRY_MQTT_TOPIC_TEMPLATE % self._donkey_name
         self._use_json_format = cfg.TELEMETRY_MQTT_JSON_ENABLE
         self._mqtt_client = MQTTClient()
         self._mqtt_client.connect(self._mqtt_broker, cfg.TELEMETRY_MQTT_BROKER_PORT)
         self._mqtt_client.loop_start()
-        print("Telemetry MQTT server connected.")
+        print("Telemetry MQTT server connected (publishing: {})".format(', '.join(self._default_inputs)))
+
+    @staticmethod
+    def filter_supported_metrics(inputs, types):
+        supported_inputs = []
+        supported_types = []
+        for ind in range(0, len(inputs if inputs is not None else 0)):
```

Comment:
Nit: You could write it as `len(inputs or [])`

Reply:
neat!